modules/atomic-widgets/controls/types/textarea-control.php

```diff
@@ -20,9 +22,16 @@ public function set_placeholder( string $placeholder ): self {
 		return $this;
 	}
 
+	public function set_value( string $value ): self {
+		$this->value = $value;
+
+		return $this;
+	}
```

Comment:
maybe adding a default value?
WDYT?
```suggestion
	public function set_value( string $value = null ): self {
		$this->value = $value ?? '';

		return $this;
	}
```